core/domain/exp_services.py

```diff
@@ -60,6 +61,7 @@
 from core.domain import taskqueue_services
 from core.domain import user_services
 from core.platform import models
+import deepdiff
```

Comment:
```suggestion

import deepdiff
```

Reply:
Shifted it above where other library imports is done.

Reply:
Done.